### PR TITLE
SSL/TLS support

### PR DESCRIPTION
**Pull request by haiodo**
Enable SSL/TLS support for both development and production HR build.

Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

### CODE REVIEW

File: products/hr-build/readme.md [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
+- SECURITY_CERT_FILE - Certificate file in PEM format.
+- SECURITY_KEY_FILE - Private key file in PEM format.
+- SECURITY_CA_FILE - Certificate Chain in PEM format.
+
+## Starting server
+
+Running server as background process could be achived with.
+
+```bash
+notail node ./server.js &
```
Comment (sumboid) [inline products/hr-build/readme.md:37]: I know that `nohup` can help to detach process but have never heard about `notail`

**Review by BykhovDenis — APPROVED** [review]

**Review by sumboid — COMMENTED** [review]
What a point to use SSL on localhost? Isn't it easier to use it only when necessary, for specific builds for instance?

**Comment by sumboid** [comment]
FYI: just checked on linux env, works fine

**Comment by haiodo** [comment]
Some stuff like auto insert of email/passwords is working differently, not sure about more stuff, so I think it is better to find all such places in dev.

**Review by sumboid — APPROVED** [review]

**Pull request merged**